README.md

```diff
@@ -1,25 +1,153 @@
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML/CSS</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Ruby on rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
-# hello-rails-react
+- **[Random message]**
+- **[Display a random message]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ismayito/hello-rails-react.git
+```
+
+### Install
+
+Install gems with:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ./bin/dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
+
+### Deployment
+
+```sh
+  git add . && git commit -m 'Your commit message' && git push
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mayito Ismail**
+
+- GitHub: [@ismayito](https://github.com/ismayito)
+- Twitter: [@IsmailMayito](https://twitter.com/IsmailMayito)
+- LinkedIn: [mayito-ismail](https://www.linkedin.com/in/mayito-ismail-2b0067178/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add_more_styling]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- [ ] If you'd like to support the project, feel free to reach me via the contacts listed at the `Authors` section
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [ ] I would like to thank [Microverse](https://www.microverse.org/?grsf=martin-9isn0t) for guiding me through this project, giving code reviews and a community of fellow coders.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please make sure that your `README` file doesn't contain any broken links, as I noticed that your `LICENSE` link is broken because the path you gave is incorrect. The `LICENSE` file doesn't exist in your repository, you should target the `MIT.md` file instead for the license in order to fix this issue. Kindly fix the broken link with the correct path and make sure that everything is fine before asking for the next review.

![image](https://github.com/ismayito/hello-rails-react/assets/66476329/990ae817-f037-4598-8ea9-0acb288437d8)